overlord/snapstate/snapstate_test.go

```diff
@@ -4987,6 +5017,9 @@ func (s *snapmgrTestSuite) TestTrySetsTryModeJailMode(c *C) {
 	s.testTrySetsTryMode(snapstate.Flags{JailMode: true}, c)
 }
 func (s *snapmgrTestSuite) TestTrySetsTryModeClassic(c *C) {
+	if !dirs.SupportsClassicConfinement() {
```

Comment:
Not sure, something I would have to look into but would prefer to do it with a follow up PR to not put more burden on this one.